lms/static/coffee/fixtures/student_admin.html

```diff
@@ -0,0 +1,112 @@
+<section id="student_admin" class="idash-section">
+<div class="student-specific-container action-type-container">
+  <h2>${_("Student-specific grade inspection")}</h2>
+  <div class="request-response-error"></div>
+  <p>
+  ${_("Specify the {platform_name} email address or username of a student here:").format(platform_name=settings.PLATFORM_NAME)}
+  <input type="text" name="student-select-progress" placeholder="${_('Student Email or Username')}">
+  </p>
+  <br>
+
+  <div class="progress-link-wrapper">
+  <p>
+    ${_("Click this link to view the student's progress page:")}
+
+    <a href="" class="progress-link" data-endpoint="${ section_data['get_student_progress_url_url'] }"> ${_("Student Progress Page")} </a>
+  </p>
```

Comment:
@ziafazal -- if you can please review the comments from @cptvitamin and ensure all HTML elements meet the accessibility recommendations he's made, that would be great, thanks!